nasa/earthdata-download
# EDD-28: Adding logo and dmg background

# Overview

### What is the feature?

Adds a app icon for Mac, Windows, and Linux and a background for the dmg screen on Mac.

### What is the Solution?

Upload the files to the right place in the application

### What areas of the application does this impact?

Icon and dmg screen

# Testing

1. Install the app on Mac
  - [ ] Confirm the install screen shows the background image
  - [ ] Confirm the application uses the icon when running
2. Install the app on Windows
  - [ ] Confirm the icon shows for the application
3. Install the app on Linux
  - [ ] Confirm the icon shows for the application

### Attachments

#### Mac App Icon
<img width="248" alt="image" src="https://github.com/nasa/earthdata-download/assets/7462238/9ce63c80-dd6d-4dac-b05a-db27afb81ff0">

#### Mac DMG Background
<img width="662" alt="image" src="https://github.com/nasa/earthdata-download/assets/7462238/c3ebceac-4faa-4ccc-b8e1-7b56e3f769ed">

### Windows Icon
![image](https://github.com/nasa/earthdata-download/assets/7462238/a35e962f-5a91-47a3-b18a-bc2445eeae75)

#### Linux Icon
![image](https://github.com/nasa/earthdata-download/assets/7462238/e1c9d8b1-ed8b-4612-af4c-b6fce0b05cae)

# Checklist
